### PR TITLE
Add PTC/Creo Parametric and PTC/Creo View

### PR DESCRIPTION
**Pull request by DonChunior**
Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - [x] PR Title: Add PTC/Creo Parametric and PTC/Creo View
  - [x] Status: please select a value from the status table at the top
  - [x] Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - [ ] Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - [x] Please mind the sorting

### CODE REVIEW

File: software/software_list_p.md [hunk 1]
```diff
@@ -182,8 +182,10 @@
 |PTC|Arbortext Editor, Styler & Publishing Engine|>8.0.0.0|Not vuln|Workaround| | | |[source](https://www.ptc.com/en/support/article/CS358998)|
 |PTC|Arena| | |Not vuln| | | |[source](https://www.ptc.com/en/documents/log4j)|
 |PTC|Axeda| |Not vuln|Workaround| | | |[source](https://www.ptc.com/en/support/article/CS358990)|
-|PTC|Axeda Platform|6.9.2| |Vulnerable| | | |[PTC Axeda Platform Apache log4j vulnerability - Incident Response](https://www.ptc.com/en/support/article/CS358990)|
+|PTC|Axeda Platform|6.9.2| |Vulnerable| | | |[source](https://www.ptc.com/en/support/article/CS358990)|
 |PTC|Creo Elements/Direct Model Manager| |Not vuln|Workaround| | | |[source](https://www.ptc.com/en/support/article/CS358965)|
+|PTC|Creo Parametric| |Not vuln| | | | |[source](https://www.ptc.com/en/support/article/CS359127)|
+|PTC|Creo View| | |Not vuln| | | |[source](https://ptc.com/log4j)|
```
Comment (certeu) [inline software/software_list_p.md:188]: The link for Creo Parametric and Creo View redirect to a login page. It's not possible confirm the "Not vuln" status. Could you please add a screenshot in the vendor-statement directory so we can have a kind of proof?

**Comment by certeu** [comment]
Hello @DonChunior, Thank you very much for your work. Could you please have a look at my comment before we could merge?

**Comment by DonChunior** [comment]
Hello @certeu,
I've added the two screenshots you asked for and linked them as source.

**Comment by certeu** [comment]
Hello @DonChunior, thanks a lot for the sreenshots, I will merge your update!

**Pull request merged**